treatment-service/config/config.go

```diff
@@ -109,12 +117,34 @@ func (c *Config) ListenAddress() string {
 	return fmt.Sprintf(":%d", c.Port)
 }
 
+func (c *Config) Validate() error {
+	validate := validator.New()
+
+	if c.MessageQueueConfig != nil {
+		if err := validate.Struct(c.MessageQueueConfig); err != nil {
+			return fmt.Errorf("invalid message queue configuration: %w", err)
+		}
+	}
+
+	if c.PollerConfig != nil {
+		if err := validate.Struct(c.PollerConfig); err != nil {
```

Comment:
I don't see any `validate` tag for `PoolerConfig`, is this still required?

Reply:
This is not required now. Removed it